meson.build

```diff
@@ -74,8 +74,9 @@ hse_dep = dependency(
         '<@0@.0.0'.format(hse_java_major_version + 1),
     ],
     default_options: [
-        'warning_level=0',
         'default_library=static',
+        'warning_level=0',
+        'werror=false',
```

Comment:
What does it do?

Reply:
Just says that hse-java should ignore any warnings generated by HSE. Since this is the hse-java build, we really only care about warnings that we can control (hse-java related warnings).